### PR TITLE
fix the assignment style

### PR DESCRIPTION
**Pull request by subaroon**
This pull request makes the following improvements to the Makefiles:

- Prefer `:=` over `=`.
- Use `:=` instead of `+=` for initial assignments.
- Remove unnecessary assignments.

### CODE REVIEW

File: tracing02-xdp-monitor/Makefile [hunk 1]
```diff
@@ -4,10 +4,10 @@
 XDP_TARGETS  := trace_prog_kern
 USER_TARGETS := trace_load_and_stats
 
-COMMON_DIR = ../common
+COMMON_DIR := ../common
 
 # Extend with another COMMON_OBJS
-COMMON_OBJS += $(COMMON_DIR)/common_user_bpf_xdp.o
+COMMON_OBJS := $(COMMON_DIR)/common_user_bpf_xdp.o
```
Comment (netoptimizer) [inline tracing02-xdp-monitor/Makefile:10]: This looks wrong, comment says this is extending the variable

Comment (subaroon) [reply tracing02-xdp-monitor/Makefile:10]: Thank you for your feedback. This line is clearly an initial assignment, so := is the correct operator to use here. It seems the comment might be misleading.

Comment (tohojo) [reply tracing02-xdp-monitor/Makefile:9]: Yup, definitely the comment that is wrong :)

**Review by tohojo — APPROVED** [review]
Thank you for the fixes!

**Pull request merged**